### DISCUSSION TITLE
Resources not loading

### DISCUSSION THREAD

**Discussion by stoatsnhoney** [elrumo/macOS_Big_Sur_icons_replacements#1482] [Help with the website]
The resource page isn't loading. I just get blank white loading screens on the article thumbnails. Clicking on the articles produces the same loading effect on that page. In my testing this has happened on both wifi and cellular data, and on my Mac, iPad, and iPhone. This happened in Safari, Chrome, and Edge. I haven't tried any other browsers/devices.
<img width="1531" alt="Screenshot 2024-03-08 at 1 42 01 PM" src="https://github.com/elrumo/macOS_Big_Sur_icons_replacements/assets/139384100/58488a32-b2fa-4301-8b97-5b1222e5e34b">

**Comment by OmnibusAnalysis** [comment]
Same issue

**Comment by spiny-lumpsucker** [comment]
It's still not loading for me either. 

In the meantime, you can check out the official [Apple Design Resources page](https://developer.apple.com/design/resources/) where you can download design templates for Mac and iOS/iPadOS icons

**Comment by shiro6x** [comment]
Same issue ever since I started using it 3 months
